boards/arm/rddrone_fmuk66/rddrone_fmuk66_defconfig

```diff
@@ -13,3 +14,9 @@ CONFIG_OSC_EXTERNAL=y
 # Enable MPU
 CONFIG_ARM_MPU=y
 CONFIG_HW_STACK_PROTECTION=y
+
+# Enable Regulators
+CONFIG_REGULATOR=y
+CONFIG_REGULATOR_FIXED=y
```

Comment:
this is redundant

Reply:
not addressed

Reply:
@gmarull isn't CONFIG_REGULATOR_FIXED dependent on CONFIG_REGULATOR when doing manual config? Is it not also required in defconfig?

Are you suggesting to remove all: CONFIG_REGULATOR_XXX from the defconfig?

Reply:
`CONFIG_REGULATOR_FIXED` is automatically enabled if `CONFIG_REGULATOR=y` and a node in DT with compatible `regulator-fixed` has its status set to `okay`.